src/it/java/net/spy/memcached/ClientBaseCase.java

```diff
@@ -151,6 +160,29 @@ protected void restoreClusterConfig(String config, Collection<NodeEndPoint> endp
 	}
   }
 
+  protected void restoreClusterConfigForTLS(int portNumber) throws Exception {
```

Comment:
Not sure why you need to add this method. Did the dynamic mode client test not work properly before?

Reply:
I will rename `restoreClusterConfigForTLS()` to `setClusterConfigForTLS()` to avoid confusion. This method is to set configuration for local server (for pretending as Elasticache cluster)